lib/fireblocks/fireblocks.go

```diff
@@ -0,0 +1,16 @@
+package fireblocks
+
+import (
+	"context"
+
+	f "github.com/omni-network/omni/lib/fireblocks/api"
+)
+
+type FireBlocks interface {
+	// CreateTransaction creates a new transaction on the FireBlocks API.
+	// We use raw signing by default
+	CreateTransaction(ctx context.Context, request f.CreateTransactionRequest) (string, error)
```

Comment:
what is the returned type? maybe make that a proper type. Also please mention in the doc what the method returns as this isn't obvious.